### ISSUE TITLE
never_convert_lossy_files should flip to convert_lossy_files

### ISSUE THREAD

**Issue by RollingStar** [beetbox/beets#3124]
### Problem

Convert: `never_convert_lossy_files` is one of the only config options I see in beets that uses negative wording. Instead of `never_convert_lossy_files: no`, the default could be `convert_lossy_files: yes`

**Comment by sampsyo** [comment]
Indeed; I've never really liked the name of this config option. I even think `convert_lossy` would be a fine name for the "reversed" version of this option.

If we change this, it will be important to preserve backwards compatibility by checking for an (undocumented) option with the original name and inverting it.
